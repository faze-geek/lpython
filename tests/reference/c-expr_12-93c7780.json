{
    "basename": "c-expr_12-93c7780",
    "cmd": "lpython --no-color --show-c {infile}",
    "infile": "tests/../integration_tests/expr_12.py",
    "infile_hash": "0d98c6e93b5c0765f6829f51bf12d6487b9fa0d1381e07e11bddbea7",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "c-expr_12-93c7780.stderr",
    "stderr_hash": "f611d9d0179a718bcca43e0c1c70ffc69afbd43c6b6a2eb982e59212",
    "returncode": 2
}