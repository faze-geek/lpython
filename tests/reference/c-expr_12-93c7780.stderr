semantic error: 'i16[:]' object does not support item assignment
 --> tests/../integration_tests/expr_12.py:4:5
  |
4 |     y[0] = i16(1)
  |     ^^^^^^^^^^^^^ 
